sonar-db/src/main/java/org/sonar/db/DefaultDatabase.java

```diff
@@ -168,6 +180,6 @@ private static void completeDefaultProperty(Properties props, String key, String
 
   @Override
   public String toString() {
-    return "Database[" + (properties != null ? properties.getProperty(SONAR_JDBC_URL) : "?") + "]";
+    return format("Database[%s]", properties != null ? properties.getProperty(SONAR_JDBC_URL) : "?");
```

Comment:
why not, but not sure it makes sense on such class.
@sns-seb do you know if there's a Java rule for that ? It could be interesting.